packages/hub/config/structure.sql

```diff
@@ -597,17 +597,6 @@ $$;
 
 ALTER FUNCTION graphile_worker.tg_jobs__notify_new_jobs() OWNER TO postgres;
 
---
--- Name: add(integer, integer); Type: FUNCTION; Schema: public; Owner: postgres
---
-
-CREATE FUNCTION public.add(integer, integer) RETURNS integer
-    LANGUAGE sql IMMUTABLE STRICT
-    AS $_$select $1 + $2;$_$;
-
-
-ALTER FUNCTION public.add(integer, integer) OWNER TO postgres;
```

Comment:
Is this because [this migration](https://github.com/cardstack/cardstack/pull/2547/files#diff-930150a7f3faea990b45d2cdfaad9876332f6fc13cec22a67cfb034969b11302R5), which includes the other `lt` `eq` and `gt` functions doesn’t have `add`? Current theory is that it was in a migration but then removed and `structure.sql` wasn’t updated.

Reply:
It seems like it’s [safe to remove](https://discord.com/channels/584043165066199050/900021750367129641/936726752259096587).

Reply:
Nice, thanks for checking!